lib/rest/Twilio.js

```diff
@@ -125,6 +126,7 @@ function Twilio(username, password, opts) {
   }
   this.edge = opts.edge || env.TWILIO_EDGE;
   this.region = opts.region || env.TWILIO_REGION;
+  this.debug = opts.debug || (/true/i).test(env.TWILIO_DEBUG);
```

Comment:
Not sure what the second half is doing. Should just be able to have `this.debug = opts.debug || env.TWILIO_DEBUG`

Reply:
`env.TWILIO_DEBUG` is originally a string. `(/true/i).test `is using regex to see if `env.TWILIO_DEBUG` includes the value true. It will return a boolean of true or false depending on if `env.TWILIO_DEBUG` includes true. If it doesn't get converted to a boolean here then anytime the user has `env.TWILIO_DEBUG` defined debugger logs will be outputted.